src/customConfigTests/src/test/kotlin/org/vaccineimpact/orderlyweb/customConfigTests/RunWorkflowTests.kt

```diff
@@ -176,11 +176,14 @@ class RunWorkflowTests : SeleniumTest()
         assertThat(submitButton.isEnabled).isFalse()
         val changelogTypes = driver.findElements(By.cssSelector("#change-type-control option"))
         assertThat(changelogTypes.count()).isEqualTo(2)
-        assertThat(changelogTypes[0].text).isEqualTo("internal")
-        assertThat(changelogTypes[1].text).isEqualTo("public")
+        assertThat(changelogTypes[0].text).contains("internal")
+        assertThat(changelogTypes[1].text).contains("public")
```

Comment:
This looks a bit fishy - is there anything in this PR that would have changed the format of this dropdown?

Reply:
This test has always failed locally for me (I get white space around internal and public) and I just got sick of having to comment it out to check the rest of the test. Always passes on BK either way though.